spec/mailers/order_mailer_spec.rb

```diff
@@ -216,4 +216,41 @@
       end
     end
   end
+
+  describe "#invoice_email" do
+    subject(:email) { described_class.invoice_email(order) }
```

Comment:
I've never seen `subject` used with a name before. It seems to behave just like `let`, so I wonder if it was intentional?

Reply:
I've seen it in the previous example.